src/scss/common/_z-layers.scss

```diff
@@ -2,6 +2,7 @@
 
 $z-layers: (
 	layer_null: 0,
```

Comment:
I don't think you need 0

I think it is enough to have -1 and 1

Reply:
I disagree with you. It's convenient in such cases: @include helpers.zIndex(layer_null, 80); , when I can add any number in non-standard cases.

Reply:
there is no 0 in my examples, there are 1, -1

you need to add number like a second parameter only if you use the same z-index for several elements, but one must have bigger z-index that another one

you still don't need the basic setting with 0

Reply:
Done